java/src/jmri/util/swing/StayOpenCheckBoxItem.java

```diff
@@ -38,15 +32,33 @@ public StayOpenCheckBoxItem(String text, boolean isSelected) {
     }
     
     private void initUI(){
-        String laf = UIManager.getLookAndFeel().getName();
-        if (!("Nimbus".equals(laf))){
+        putClientProperty("CheckBoxMenuItem.doNotCloseOnMouseClick", Boolean.TRUE);
```

Comment:
What does this line does and should it be there for all L&Fs? I'm not against it, I'm asking to understand the code.

Reply:
Hi @danielb987 

See https://docs.oracle.com/javase/9/docs/api/javax/swing/JCheckBoxMenuItem.html
This seems to work for Java LAF's > Java 9 ( including Apple LAF ) ,
This is a change from Java 8
https://docs.oracle.com/javase/8/docs/api/javax/swing/JCheckBoxMenuItem.html

This property is not present in Java 8 hence the fix.
Tested with Java 8 + Java 11 with various LAFs.

Reply:
Thanks for the explanation.